substratevm/src/com.oracle.svm.test/src/hello/Hello.java

```diff
@@ -178,4 +180,28 @@ private static void inlineTailRecursion(int n) {
         }
         inlineTailRecursion(n - 1);
     }
+
+    @NeverInline("For testing purposes")
+    private static void noInlineManyArgs(int i0, int i1, int i2, int i3, int i4, int i5, int i6, int i7, int i8,
+                    float f0, float f1, float f2, float f3, float f4, float f5, float f6, float f7, float f8) {
```

Comment:
What would currently happen with other primitive types like `boolean` or `double`?

Reply:
Nothing exciting I believe. All arguments occupy a register or an 8 byte stack slot.However, I could (or, more correctly, should) throw in some extra arguments of those types to check this works ok.

Reply:
Yes please add other constants of primitive types here as well.

Reply:
Fixed